src/main/java/com/microsoft/azure/relay/ClientWebSocket.java

```diff
@@ -289,7 +289,6 @@ else if (data instanceof String){
 			} else {
 				this.session.close();
 			}
-			((LifeCycle) this.container).stop();
```

Comment:
>((LifeCycle) this.container).stop(); [](start = 3, length = 36)

Moving this to the @OnClose block below so ClientWebSockets that close passively will terminate its running threads too.